src/init.cpp

```diff
@@ -358,6 +358,13 @@ void SetupServerArgs()
     gArgs.AddArg("-minimumchainwork=<hex>", strprintf("Minimum work assumed to exist on a valid chain in hex (default: %s, testnet: %s)", defaultChainParams->GetConsensus().nMinimumChainWork.GetHex(), testnetChainParams->GetConsensus().nMinimumChainWork.GetHex()), true, OptionsCategory::OPTIONS);
     gArgs.AddArg("-par=<n>", strprintf("Set the number of script verification threads (%u to %d, 0 = auto, <0 = leave that many cores free, default: %d)",
         -GetNumCores(), MAX_SCRIPTCHECK_THREADS, DEFAULT_SCRIPTCHECK_THREADS), false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-mitigatespectre", strprintf("Attempt to enable per process spectre mitigations (default: %u%s)", DEFAULT_MITIGATE_SPECTRE, (
+#ifdef ENABLE_WALLET
+    DEFAULT_MITIGATE_SPECTRE ? " or 0 if -disablewallet set" : "0"
```

Comment:
This will turn into `00` if `!DEFAULT_MITIGATE_SPECTE`, I think?

Reply:
debug help shows with wallet enabled:  
```
  -mitigatespectre
       Attempt to enable per process spectre mitigations (default: 1 or 0 if
       -disablewallet set)
```

Reply:
If you set `DEFAULT_MITIGATE_SPECTRE` to `false` it will say `00` I think.

Reply:
yeah, it only works because `DEFAULT_MITIGATE_SPECTRE` is controlled by `ENABLE_WALLET`

Reply:
I don't think it should rely on that.

And it does give the following output for `-disable-wallet` (at configure level) so it's broken regardless:
```
  -mitigatespectre
       Attempt to enable per process spectre mitigations (default: 00)
```